src/components/NewFormTodo/NewFormTodo.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FormEvent } from 'react';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  title: string,
+  handleTitleTodo: (newTitle: string) => void,
+  addTodo: (titleTodo: string) => void,
+  tempTodo: Todo | null,
+};
+
+export const NewFormTodo: React.FC<Props> = ({
+  title,
+  handleTitleTodo,
+  addTodo,
+  tempTodo,
+}) => {
+  const isInputVisible = !!tempTodo;
+  const handleSubmit = (event: FormEvent) => {
+    event.preventDefault();
+
+    addTodo(title);
+  };
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <input
+        type="text"
+        className="todoapp__new-todo"
+        placeholder="What needs to be done?"
+        value={title}
+        onChange={({ target }) => {
+          handleTitleTodo(target.value.trim());
```

Comment:
If you use trim there, you can't write a few words with space for title. Better to use trim when call addTodo.

Reply:
great advice)